src/esp32cam.h

```diff
@@ -29,6 +29,103 @@ class CameraClass
   bool
   changeResolution(const Resolution& resolution, int sleepFor = 500);
 
+  /**
+  * must be -2 to +2, default 0
```

Comment:
This isn't proper Doxygen syntax.

Reply:
Fixed with commit -a -amend, please verify if it is correct now - have not used doxygen yet